ninja-test-utilities/src/main/java/ninja/NinjaFluentLeniumTest.java

```diff
@@ -20,15 +20,20 @@
 import org.junit.After;
 import org.junit.Before;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.htmlunit.HtmlUnitDriver;
 
 import com.google.inject.Injector;
 
 import ninja.utils.NinjaTestServer;
 
 public abstract class NinjaFluentLeniumTest extends FluentTest {
 
-	public WebDriver webDriver = new HtmlUnitDriver();
+  /**
+   * @deprecated The driver should be access via {@link #getDriver()}.
+   * Null variable left for deprecation notice only.
+   * @see #getDriver()
+   */
+  @Deprecated
+	public WebDriver webDriver;
```

Comment:
Already a mistake in the codebase... but would you mind fixing the tabs to space? thanks!